goss.yaml

```diff
@@ -0,0 +1,81 @@
+command:
```

Comment:
please move the goss.yaml and goss_wait.yaml files into the `3.1.0` directory and adjust the justfile targets to work with these change

this will allow us to develop version specific tests for new rock versions in the future